### ISSUE TITLE
Error of the weight weight_of_ood_l2

### ISSUE THREAD

**Issue by aithuuuus** [Baichenjia/PBRL#1]
[CODE](https://github.com/Baichenjia/PBRL/blob/f2aac62d8780f8725054b326b728ebb41d17e003/d4rl/rlkit/torch/sac/pevi.py#L272H)
```			
               cat_qf_ood_target = torch.cat([
				torch.maximum(qf_curr_actions_all[qf_index] - weight_of_ood_l2 * ucb_curr_actions, torch.zeros(1).cuda()),
				torch.maximum(qf_next_actions_all[qf_index] - 0.1 * ucb_next_actions, torch.zeros(1).cuda())], 0)
```
Hi, I have a small question about this line of code, should the 0.1 be changed to `weight_of_ood_l2` ($\beta^{ood}$, just like the line above)?

**Comment by Baichenjia** [comment]
I find regularizing the Q-value of the next action does not contribute much to the performance. So using 0.1 or weight_of_ood_l2 does not make much difference in the final performance. Further, you can try to remove the second regularization completely.

**Comment by aithuuuus** [comment]
Thank you for your reply.

**Comment by aithuuuus** [comment]
It seems that this is because d4rl's dataset is collected sequentially, obs and next obs are mostly overlapping, and due to the overlap, the loss of next obs is covered by the previous one.
